example/streamed-list-objects-local/streamedListObjectsLocal.mjs

```diff
@@ -0,0 +1,62 @@
+import { ClientConfiguration, OpenFgaClient, ConsistencyPreference } from "../../dist/index.js";
```

Comment:
I don't think we need two examples for this, let's stick to this one only,

Also can you add the package.json for this example?